src/help/zaphelp/contents/ui/dialogs/manageaddons.html

```diff
@@ -29,6 +29,11 @@
 You can install any of the add-ons listed - typically add-ons are dynamically added to the ZAP UI
 as soon as you install them.
 
+<H2>Manual loading</H2>
+When working in a disconnected environment or debugging add-ons, you may want to manually download and install an add-on. <br/>
```

Comment:
Could you provide some details about `debugging add-ons`? Is this from a POV of a dev or a user? (If the former there are better ways to debug/test the add-ons.)

Reply:
Tbh I was thinking of a devs perspective, or someone who is testing an unpublished version of an add-on. If there really are better ways to test and debug, then I’d happily hear about those too :P

Reply:
If it's simply to test the add-on, installing as already mentioned is the way to go. For debugging you would want to attach your debugger/IDE into ZAP, that way you can debug/breakpoint/change the ZAP/add-on on the fly (granted not all changes can be applied it does reduce the number of times the add-on needs to be built/deployed).

I'd leave out the `debugging add-ons` since this is the user guide, we can set up a `zaproxy` wiki page for that (or change this one? https://github.com/zaproxy/zaproxy/wiki/Setting-up-and-developing-new-plugins).